### PR TITLE
TTK-12915 Add read only roles

### PR DESCRIPTION
**Pull request by rubenrua**

### CODE REVIEW

File: src/Pumukit/NewAdminBundle/Resources/views/Person/listrelationro.html.twig [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{% if app.user and is_granted(constant('Pumukit\\SchemaBundle\\Security\\Permission::ACCESS_PEOPLE')) %}
+<table>
+  <tdody>
```
Comment (Yurujai) [inline src/Pumukit/NewAdminBundle/Resources/views/Person/listrelationro.html.twig]: change <tdody> to <tbody>

Comment (Yurujai) [inline src/Pumukit/NewAdminBundle/Resources/views/Person/listrelationro.html.twig]: "<tdody>" to "<tbody>"

File: src/Pumukit/NewAdminBundle/Resources/views/Person/listrelationro.html.twig [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    {% for person in people %}
+    <tr>
+      <td><ul><li></li></ul></td>
```
Comment (Yurujai) [inline src/Pumukit/NewAdminBundle/Resources/views/Person/listrelationro.html.twig]: why a empty list ? and you can use colspan to write some td

File: src/Pumukit/NewAdminBundle/Resources/views/Person/listrelationro.html.twig (hunk 1 above)
Comment (rubenrua) [inline src/Pumukit/NewAdminBundle/Resources/views/Person/listrelationro.html.twig]: Copy&Pase.

Fixed all.

**Comment by Yurujai** [comment]
thank you

**Pull request merged**